stream_alert/alert_processor/output_base.py

```diff
@@ -19,12 +19,14 @@
 import os
 import tempfile
 import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
```

Comment:
Please just import urllib3 directly instead of accessing through the requests package. Example:

```
import urllib3
...
urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```
Unless I'm missing something and the method you're using is the only way this will work? Can you also add a note as to why this is necessary?

Reply:
Will do.  The change is related to the requests library generating warning messages for every request that is made with the certificate verification disabled (verify=False).  The code above is to suppress those warnings, as the streamalert logs were full of these warnings.  A single alert to Phantom or Jira could generate 3 of these warnings per alert, so it can get pretty noisy.

Unfortunately, it looks like this is the only way to suppress the errors as we are using an older version of requests:
 [v2.13.0](https://github.com/airbnb/streamalert/blob/master/requirements.txt#L71).  With the most recent version of requests (v2.18.4), it seems like what you suggested would actually work.  I am referencing this SO [post](https://stackoverflow.com/a/28002687).

Reply:
Thanks for the clarification - in that case, let's just bump our `requests` min requirement to that version and switch to the aforementioned way of disabling the warnings.